src/server/server-quiz/services/quiz-manager.js

```diff
@@ -1,10 +1,63 @@
+const urlArray = require("../clients/api-constants");
 const { Question } = require("../../db/models");
+const triviaClient = require("../clients/trivia-client");
 
 async function getAllQuestions() {
-  const questions = await Question.findAll();
+  const personalQuestions = await Question.findAll();
```

Comment:
where do you set the amount of questions?

Reply:
in line 25

Reply:
For now I didn't set it yet